pkg/localstore/disaster_recovery.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2021 The go-ethereum Authors
```

Comment:
s the header still relevant (also in other files)?

Reply:
not sure, @acud ?